cmd/epithet-oidc-plugin/epithet-oidc-plugin.go

```diff
@@ -48,6 +78,16 @@ func run(cc *cobra.Command, args []string) error {
 
 	ctx := context.Background()
 
+	var refreshToken string
+	scanner := bufio.NewScanner(os.Stdin)
+	for scanner.Scan() {
+		refreshToken = scanner.Text()
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.WithError(err).Warn("failed to read from stdin")
```

Comment:
Is this really a warning, or is it an error?

Reply:
I don't it's a warning, previous state is optional, if there is a problem reading previous state, it should not break.

Reply:
It is being logged as a Warn